apps/charterafrica/src/pages/api/v1/resources/[action].page.js

```diff
@@ -4,23 +4,24 @@ import payload from "@/charterafrica/lib/payload";
 async function datasets(req, res) {
   const {
     query: {
```

Comment:
`[action].page.js` is only for actions i.e. update, create, resources. What you're doing here is just a plain data retrieval API endpoint.

This should be in `datasets.page.js`. It doesn't share anything with `[actions].page.js`. Otherwise, we're going to make `[actions].page.js` very complex.